apps/sertra/pages/server/[id].tsx

```diff
@@ -44,6 +46,13 @@ export const getStaticProps: GetStaticProps<
 const ServerDetail: React.FC<{ detail_listing: ServerListingDetailData }> = ({
   detail_listing,
 }) => {
+  const [showPopup, setShowPopup] = useState(false);
+  const togglePopup = () => {
+    setShowPopup(!showPopup);
+    if (!showPopup) {
```

Comment:
Have you disabled the modal intentionally based on the discord discussion? Just making sure this works as you think it does, since at least for me the modal doesn't currently open when the button is pressed (since the value of `showPopup` hasn't yet be updated by the line above when this line is executed. If this was intentional, then maybe it should be at least documented, but I'd prefer to comment out this whole `if` block.